build/scripts/test-build-correctness.ps1

```diff
@@ -25,15 +25,15 @@ try {
     # Need to parse out the current NuGet package version of Structured Logger
     $structuredLoggerPath = Join-Path (Get-PackageDir "Microsoft.Build.Logging.StructuredLogger") "lib\net46\StructuredLogger.dll"
     $configDir = Join-Path $binariesDir $config
-    $logPath = Join-Path $configDir "build.xml"
+    $logPath = Join-Path $configDir "roslyn.buildlog"
```

Comment:
Can you update our groovy file to make sure we include buildlog files in the archiving process? Otherwise we won't be able to dig into this during failed jobs.

https://github.com/dotnet/roslyn/blob/master/netci.groovy#L21